pages/admin.tsx

```diff
@@ -16,12 +19,30 @@ const Admin: ComponentWithAuth = () => {
         <title>creativity tracker</title>
       </Head>
       {timeclock}
+      <Button
+        onClick={() => fetch("/api/storeNlpData")}
+        variant="outlined"
+        sx={{ alignSelf: "flex-start", m: 5 }}
+      >
+        run store nlp data cron job
```

Comment:
added two testing tools to the admin dashboard. this one runs the api call on line 23. this way we can test the cron job on develop because it will only be triggered nightly in the production environment, so only when we have pushed code to the main branch